spec/support/mockResponseData.json

```diff
@@ -1,5 +1,58 @@
 {
-    "bid": [],
-    "pass": [],
-    "deals": []
+  "bid": {
```

Comment:
Are the `"bid", "pass" and "deals"` test cases or are they actual part of the bid response? 
(Will the bid response be something like `{"1163":[{"a": 1}]}` or `{"bid": {"1163": [{"a": 1}]}}`?)

Reply:
@ix-certification they are actual part of the response. Example here:
```
{"bid":{"34364":[{"id":338817,"height":600,"width":300,"price":0.004165,"format":"is","adm":"<script>(function setup(e,n,i,d){function t(){return l&&l.requested}function r(){return l&&l.initialized}function o(){return!!e.Jpx&&e.Jpx.JAM}function a(e,n){var i=m.createElement(\"script\");i.onload=n,i.src=e,m.head.appendChild(i)}function s(){l=e.jPAM=e.Jpx.JAM.instance({plugins:[\"bidder\"]}),c()}function c(){var i=e.jPAM=e.jPAM||{};i.cmd=i.cmd||[],i.cmd.push(function(){var i=(e.jPAM=e.jPAM||{}).getPlugin(\"bidder\");i.loadBid(n),i.renderAd(n.sourceId)}),i.processCmd&&i.processCmd()}function u(){return r()?(l.hasPlugin(\"bidder\")||l.addPlugin(\"bidder\"),c()):o()?s():t()&&!o()?f-- >0&&setTimeout(u,36):(l.requested=!0,void a(i.libUrl,s))}function p(n){var i=w.createElement(\"img\");w.body.appendChild(i),i.setAttribute(\"src\",e.location.protocol+n)}var l=e.jPAM=e.jPAM||{},m=e.document,w=window.document,f=300;p(function(n){var d=n.server;return i.impTD+\"?rid=\"+d.ruid+\"&sid=\"+d.suid+\"&uid=\"+d.uuid+\"&vr=\"+d.adpVer+\"&ru=\"+encodeURIComponent((e.location.href||\"\").replace(/#/g,\"\"))+\"&tt=\"+ +new Date+\"&siw=\"+n.settings.site.width+\"&sh=\"+window.screen.availHeight+\"&sw=\"+window.screen.availWidth+\"&wh=\"+e.innerHeight+\"&ww=\"+e.innerWidth+\"&an=\"+d.app+\"&vn=\"+d.cluster+\"&sd=\"+d.subdomain+\"&_c=a\"+Math.random().toString(36).substring(7)+ +new Date+\"&et=&aid=\"+n.adId+\"&said=\"+n.sourceId+\"&ei=\"+encodeURIComponent(n.placement)+\"&fc=\"+n.format+\"&sp=\"+n.ssp+\"&at=bidder&cid=&ist=\"+n.isTest+\"&mg=&dl=&dlt=&ev=&vt=&zid=\"+n.zoneId+\"&dr=\"+(+new Date-d.registeredAt)+\"&di=\"+(n.dealId||\"\")+\"&pr=\"+window.btoa(\"\"+parseInt(1e3*n.cpm))+\"&cw=&ch=&nt=&st=&jp=&ty=bim\"}(n)),p(i.imp+\"/?zone=\"+n.zoneId+\"&adid=\"+n.adId+\"&rid=\"+n.server.ruid+\"&i=\"+ +new Date),function(){d&&(n.prerendered=d(decodeURIComponent(n.creative.html),n.sourceId,n.format,n.inits.inline))}(),u()}(window.top, {\"sourceId\":338817,\"adId\":199139,\"ssp\":22,\"format\":\"is\",\"height\":600,\"width\":300,\"inventory\":100,\"placement\":\"538883878\",\"views\":0,\"viewsTime\":0,\"boxTimeout\":\"20\",\"animations\":1,\"init\":\"\",\"trackingUrl\":\"\",\"trackingUrl2\":\"\",\"tag\":{\"zoneid\":34364,\"subdomain\":\"ads\",\"trackingStatus\":1,\"options\":{\"animate\":1},\"debug\":true,\"fix_pd\":false,\"channel\":\"xx\",\"country\":\"nl\",\"sitewidth\":960,\"fallback\":\"\",\"minify\":true},\"cpm\":0.004165,\"html\":\"%3Cdiv%20id%3D'beacon_YvT42guWDDB6QSHF4gUQaVyEWQ'%20style%3D'position%3A%20absolute%3B%20left%3A%200px%3B%20top%3A%200px%3B%20visibility%3A%20hidden%3B'%3E%3Cimg%20src%3D'http%3A%2F%2Fdelivery-europe-west-1.openx.net%2Fw%2F1.0%2Fri%3Fph%3D26e53f82-d199-49df-9eca-7b350c0f9646%26ts%3D1fHJpZD1mYTI1NWUwZi01MTU0LTQ4MDQtYTQ1Ni1hNzgyYTI5MTc0YTV8cnQ9MTUxMjY1MTg1OXxhdWlkPTUzODg4Mzg3OHxhdW09RE1JRC5XRUJ8YXVwZj1kaXNwbGF5fHNpZD01Mzg4ODM4Nzd8cHViPTUzNzExNjc0NHxwYz1VU0R8cmFpZD1kYWNiOTc5Mi1mNTVlLTRiZDQtYjJmZi1lODk5NjA5NDFhNTF8cnM9MXxhaWQ9NTM5MTM3NDg5fHQ9NHxhcz0zMDB4NjAwfGxpZD01MzkwOTc2NjN8b2lkPTUzODg4MzIyMHxwPTEwfHByPTEwfGF0Yj0xMHxhZHY9NTM3MTE0ODM4fGFjPVVTRHxwbT1QUklDSU5HLkNQTXxibT1CVVlJTkcuSE9VU0V8bGl0PUh8dXI9MW8wMEdNZXI3NQ'%2F%3E%3C%2Fdiv%3E%3CSCRIPT%20TYPE%3D%22application%2Fjavascript%22%20SRC%3D%22https%3A%2F%2Fpixel.adsafeprotected.com%2Frjss%2Fst%2F113340%2F19764116%2Fskeleton.js%22%3E%3C%2FSCRIPT%3E%0A%3CNOSCRIPT%3E%3CIMG%20SRC%3D%22https%3A%2F%2Fpixel.adsafeprotected.com%2Frfw%2Fst%2F113340%2F19764115%2Fskeleton.gif%22%20BORDER%3D0%20WIDTH%3D1%20HEIGHT%3D1%20ALT%3D%22%22%3E%3C%2FNOSCRIPT%3E%0A%3Cscript%20type%3D%22text%2Fjavascript%22%3E%0A%20%20%20%20window.jpx_responsive%20%3D%20true%3B%0A%20%20%20%20window.top.jp_creative_vars%20%3D%20window.jp_creative_vars%20%3D%20%7B%0A%20%20%20%20%20%20%20%20%22landingpage%22%3A%20%22https%3A%2F%2Fwww.global.adidas.com%2F%22%2C%0A%20%20%20%20%20%20%20%20%22clickmacro%22%3A%20%22http%3A%2F%2Fox-d.justpremium.com%2Fw%2F1.0%2Frc%3Fts%3D1fHJpZD1mYTI1NWUwZi01MTU0LTQ4MDQtYTQ1Ni1hNzgyYTI5MTc0YTV8cnQ9MTUxMjY1MTg1OXxhdWlkPTUzODg4Mzg3OHxhdW09RE1JRC5XRUJ8YXVwZj1kaXNwbGF5fHNpZD01Mzg4ODM4Nzd8cHViPTUzNzExNjc0NHxwYz1VU0R8cmFpZD1kYWNiOTc5Mi1mNTVlLTRiZDQtYjJmZi1lODk5NjA5NDFhNTF8cnM9MXxhaWQ9NTM5MTM3NDg5fHQ9NHxhcz0zMDB4NjAwfGxpZD01MzkwOTc2NjN8b2lkPTUzODg4MzIyMHxwPTEwfHByPTEwfGF0Yj0xMHxhZHY9NTM3MTE0ODM4fGFjPVVTRHxwbT1QUklDSU5HLkNQTXxibT1CVVlJTkcuSE9VU0V8bGl0PUh8dXI9MW8wMEdNZXI3NQ%26r%3D%22%0A%20%20%20%20%7D%3B%0A%0A%3C%2Fscript%3E%0A%3Cscript%20type%3D%22text%2Fjavascript%22%20src%3D%22https%3A%2F%2Fd2nvliyzbo36lk.cloudfront.net%2Fapi%2Fjcl.v2.min.js%3Fv%3D1%22%3E%3C%2Fscript%3E%0A%3Cscript%20type%3D%22text%2Fjavascript%22%3E%0A%20%20%20%20var%20clickurl%20%3D%20%22http%3A%2F%2Fox-d.justpremium.com%2Fw%2F1.0%2Frc%3Fts%3D1fHJpZD1mYTI1NWUwZi01MTU0LTQ4MDQtYTQ1Ni1hNzgyYTI5MTc0YTV8cnQ9MTUxMjY1MTg1OXxhdWlkPTUzODg4Mzg3OHxhdW09RE1JRC5XRUJ8YXVwZj1kaXNwbGF5fHNpZD01Mzg4ODM4Nzd8cHViPTUzNzExNjc0NHxwYz1VU0R8cmFpZD1kYWNiOTc5Mi1mNTVlLTRiZDQtYjJmZi1lODk5NjA5NDFhNTF8cnM9MXxhaWQ9NTM5MTM3NDg5fHQ9NHxhcz0zMDB4NjAwfGxpZD01MzkwOTc2NjN8b2lkPTUzODg4MzIyMHxwPTEwfHByPTEwfGF0Yj0xMHxhZHY9NTM3MTE0ODM4fGFjPVVTRHxwbT1QUklDSU5HLkNQTXxibT1CVVlJTkcuSE9VU0V8bGl0PUh8dXI9MW8wMEdNZXI3NQ%26r%3Dhttps%3A%2F%2Fwww.global.adidas.com%2F%22%3B%0A%20%20%20%20try%20%7B%0A%20%20%20%20%20%20%20%20window.top.jpx_ad_config%20%3D%20%7B%0A%20%20%20%20%20%20%20%20%20%20%20%20format%3A%20%22is%22%2C%0A%20%20%20%20%20%20%20%20%20%20%20%20clickURL%3A%20clickurl%2C%0A%20%20%20%20%20%20%20%20%20%20%20%20callback%3A%20(typeof%20creativeCallback%20!%3D%3D%20'undefined')%20%3F%20creativeCallback%20%3A%20function()%20%7B%7D%2C%0A%20%20%20%20%20%20%20%20%20%20%20%20layers%3A%20%7B%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20global%3A%20%7B%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20width%3A%20300%2C%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20height%3A%20600%2C%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20url%3A%20%22https%3A%2F%2Fd2nvliyzbo36lk.cloudfront.net%2Fapi%2FPE%2FAdidas_300x600%2Ftest%2F1504615327%2Fout-index.js%22%2C%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20type%3A%20%22zip%22%2C%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20script%3A%20%22https%3A%2F%2Fd2nvliyzbo36lk.cloudfront.net%2Fapi%2FPE%2FAdidas_300x600%2Ftest%2F1504615327%2Fout-index.js%22%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%20%20%20%20%20%20%7D%2C%0A%20%20%20%20%20%20%20%20%20%20%20%20layout%3A%20%7B%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20responsive%3A%20true%2C%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20type%3A%20%22full%22%0A%20%20%20%20%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%20%20%7D%3B%0A%20%20%20%20%7D%20catch%20(e)%20%7B%7D%0A%0A%3C%2Fscript%3E%0A%3Cscript%20type%3D%22text%2Fjavascript%22%20src%3D%22https%3A%2F%2Fd2nvliyzbo36lk.cloudfront.net%2Fapi%2FPE%2FAdidas_300x600%2Ftest%2F1504615327%2Fout-index.js%22%3E%3C%2Fscript%3E\",\"adunitid\":538883878,\"currency\":\"EUR\",\"options\":{\"options\":{\"vast\":{\"closeenable\":2,\"autoclose\":1,\"buffering\":2,\"adtimeout\":20,\"autoclosetimeout\":2,\"initialvolume\":20,\"clickaction\":0,\"adcap\":0}},\"mobile\":{\"animations\":{\"show\":0,\"close\":0}},\"iframe\":{\"inDapIF\":{\"status\":1,\"value\":\"1\"}}},\"isTest\":0,\"reqId\":34364,\"settings\":{\"site\":{\"width\":960},\"product\":{}},\"inits\":{\"inline\":\"[[\\\"#footer\\\",\\\"before_sibling\\\"]]\",\"site\":\"var me = this;\",\"product\":\"this.addCallback(\\\"init\\\", function () {\\n    this.products = this.products || {};\\n    this.products.is = this.products.is || {};\\n    this.products.is.isAutoInsertable = this.products.is.isAutoInsertable || function () {\\n        return Jpx.dom.isValidGoogleIframe();\\n    };\\n});\"},\"server\":{\"device\":\"phone\",\"adpVer\":\"v2.49.146\",\"debug\":true,\"ruid\":\"r-15e95ebd-dd42-49d5-9a1c-2b7da55dad9a-4928718-919062828\",\"suid\":\"s-48370bd0-7e21-419e-8a5b-9a560ce6c7e6-4946128-572666345\",\"uuid\":\"u-cc39fe4c-184f-4f69-9cb4-f6edfeced74a-4826121-527729489\",\"app\":\"app20\",\"cluster\":\"amsterdam\",\"subdomain\":\"pre\",\"zone\":34364,\"registeredAt\":1512651858934},\"zoneId\":34364,\"creative\":{\"html\":\"%3Cdiv%20id%3D'beacon_YvT42guWDDB6QSHF4gUQaVyEWQ'%20style%3D'position%3A%20absolute%3B%20left%3A%200px%3B%20top%3A%200px%3B%20visibility%3A%20hidden%3B'%3E%3Cimg%20src%3D'http%3A%2F%2Fdelivery-europe-west-1.openx.net%2Fw%2F1.0%2Fri%3Fph%3D26e53f82-d199-49df-9eca-7b350c0f9646%26ts%3D1fHJpZD1mYTI1NWUwZi01MTU0LTQ4MDQtYTQ1Ni1hNzgyYTI5MTc0YTV8cnQ9MTUxMjY1MTg1OXxhdWlkPTUzODg4Mzg3OHxhdW09RE1JRC5XRUJ8YXVwZj1kaXNwbGF5fHNpZD01Mzg4ODM4Nzd8cHViPTUzNzExNjc0NHxwYz1VU0R8cmFpZD1kYWNiOTc5Mi1mNTVlLTRiZDQtYjJmZi1lODk5NjA5NDFhNTF8cnM9MXxhaWQ9NTM5MTM3NDg5fHQ9NHxhcz0zMDB4NjAwfGxpZD01MzkwOTc2NjN8b2lkPTUzODg4MzIyMHxwPTEwfHByPTEwfGF0Yj0xMHxhZHY9NTM3MTE0ODM4fGFjPVVTRHxwbT1QUklDSU5HLkNQTXxibT1CVVlJTkcuSE9VU0V8bGl0PUh8dXI9MW8wMEdNZXI3NQ'%2F%3E%3C%2Fdiv%3E%3CSCRIPT%20TYPE%3D%22application%2Fjavascript%22%20SRC%3D%22https%3A%2F%2Fpixel.adsafeprotected.com%2Frjss%2Fst%2F113340%2F19764116%2Fskeleton.js%22%3E%3C%2FSCRIPT%3E%0A%3CNOSCRIPT%3E%3CIMG%20SRC%3D%22https%3A%2F%2Fpixel.adsafeprotected.com%2Frfw%2Fst%2F113340%2F19764115%2Fskeleton.gif%22%20BORDER%3D0%20WIDTH%3D1%20HEIGHT%3D1%20ALT%3D%22%22%3E%3C%2FNOSCRIPT%3E%0A%3Cscript%20type%3D%22text%2Fjavascript%22%3E%0A%20%20%20%20window.jpx_responsive%20%3D%20true%3B%0A%20%20%20%20window.top.jp_creative_vars%20%3D%20window.jp_creative_vars%20%3D%20%7B%0A%20%20%20%20%20%20%20%20%22landingpage%22%3A%20%22https%3A%2F%2Fwww.global.adidas.com%2F%22%2C%0A%20%20%20%20%20%20%20%20%22clickmacro%22%3A%20%22http%3A%2F%2Fox-d.justpremium.com%2Fw%2F1.0%2Frc%3Fts%3D1fHJpZD1mYTI1NWUwZi01MTU0LTQ4MDQtYTQ1Ni1hNzgyYTI5MTc0YTV8cnQ9MTUxMjY1MTg1OXxhdWlkPTUzODg4Mzg3OHxhdW09RE1JRC5XRUJ8YXVwZj1kaXNwbGF5fHNpZD01Mzg4ODM4Nzd8cHViPTUzNzExNjc0NHxwYz1VU0R8cmFpZD1kYWNiOTc5Mi1mNTVlLTRiZDQtYjJmZi1lODk5NjA5NDFhNTF8cnM9MXxhaWQ9NTM5MTM3NDg5fHQ9NHxhcz0zMDB4NjAwfGxpZD01MzkwOTc2NjN8b2lkPTUzODg4MzIyMHxwPTEwfHByPTEwfGF0Yj0xMHxhZHY9NTM3MTE0ODM4fGFjPVVTRHxwbT1QUklDSU5HLkNQTXxibT1CVVlJTkcuSE9VU0V8bGl0PUh8dXI9MW8wMEdNZXI3NQ%26r%3D%22%0A%20%20%20%20%7D%3B%0A%0A%3C%2Fscript%3E%0A%3Cscript%20type%3D%22text%2Fjavascript%22%20src%3D%22https%3A%2F%2Fd2nvliyzbo36lk.cloudfront.net%2Fapi%2Fjcl.v2.min.js%3Fv%3D1%22%3E%3C%2Fscript%3E%0A%3Cscript%20type%3D%22text%2Fjavascript%22%3E%0A%20%20%20%20var%20clickurl%20%3D%20%22http%3A%2F%2Fox-d.justpremium.com%2Fw%2F1.0%2Frc%3Fts%3D1fHJpZD1mYTI1NWUwZi01MTU0LTQ4MDQtYTQ1Ni1hNzgyYTI5MTc0YTV8cnQ9MTUxMjY1MTg1OXxhdWlkPTUzODg4Mzg3OHxhdW09RE1JRC5XRUJ8YXVwZj1kaXNwbGF5fHNpZD01Mzg4ODM4Nzd8cHViPTUzNzExNjc0NHxwYz1VU0R8cmFpZD1kYWNiOTc5Mi1mNTVlLTRiZDQtYjJmZi1lODk5NjA5NDFhNTF8cnM9MXxhaWQ9NTM5MTM3NDg5fHQ9NHxhcz0zMDB4NjAwfGxpZD01MzkwOTc2NjN8b2lkPTUzODg4MzIyMHxwPTEwfHByPTEwfGF0Yj0xMHxhZHY9NTM3MTE0ODM4fGFjPVVTRHxwbT1QUklDSU5HLkNQTXxibT1CVVlJTkcuSE9VU0V8bGl0PUh8dXI9MW8wMEdNZXI3NQ%26r%3Dhttps%3A%2F%2Fwww.global.adidas.com%2F%22%3B%0A%20%20%20%20try%20%7B%0A%20%20%20%20%20%20%20%20window.top.jpx_ad_config%20%3D%20%7B%0A%20%20%20%20%20%20%20%20%20%20%20%20format%3A%20%22is%22%2C%0A%20%20%20%20%20%20%20%20%20%20%20%20clickURL%3A%20clickurl%2C%0A%20%20%20%20%20%20%20%20%20%20%20%20callback%3A%20(typeof%20creativeCallback%20!%3D%3D%20'undefined')%20%3F%20creativeCallback%20%3A%20function()%20%7B%7D%2C%0A%20%20%20%20%20%20%20%20%20%20%20%20layers%3A%20%7B%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20global%3A%20%7B%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20width%3A%20300%2C%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20height%3A%20600%2C%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20url%3A%20%22https%3A%2F%2Fd2nvliyzbo36lk.cloudfront.net%2Fapi%2FPE%2FAdidas_300x600%2Ftest%2F1504615327%2Fout-index.js%22%2C%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20type%3A%20%22zip%22%2C%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20script%3A%20%22https%3A%2F%2Fd2nvliyzbo36lk.cloudfront.net%2Fapi%2FPE%2FAdidas_300x600%2Ftest%2F1504615327%2Fout-index.js%22%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%20%20%20%20%20%20%7D%2C%0A%20%20%20%20%20%20%20%20%20%20%20%20layout%3A%20%7B%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20responsive%3A%20true%2C%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20type%3A%20%22full%22%0A%20%20%20%20%20%20%20%20%20%20%20%20%7D%0A%20%20%20%20%20%20%20%20%7D%3B%0A%20%20%20%20%7D%20catch%20(e)%20%7B%7D%0A%0A%3C%2Fscript%3E%0A%3Cscript%20type%3D%22text%2Fjavascript%22%20src%3D%22https%3A%2F%2Fd2nvliyzbo36lk.cloudfront.net%2Fapi%2FPE%2FAdidas_300x600%2Ftest%2F1504615327%2Fout-index.js%22%3E%3C%2Fscript%3E\"}}, {\"libUrl\":\"//cdn-cf.justpremium.com/js/v2.49.146/jpx.js\",\"imp\":\"//pre.ads.justpremium.com/v/1.4/t/impression\",\"impTD\":\"//emea-v3.tracking.justpremium.com/tracking.gif\"}, function Inline(e,n,t,r){function o(e){var n,t={};return l.forEach(function(r){t[r]=function(){var t=arguments;n?n[r].apply(n,t):e.jpxOnReady(function(e){n=e,e[r].apply(e,t)})}}),t}function i(e,n,t,r){for(var i,d=p.createElement(e);i=t.pop();)d.setAttribute(i[0],i[1]);if(\"IFRAME\"===d.tagName.toUpperCase()?a=function(){var e=(d.contentWindow||d.contentDocument.document||d.contentDocument||{}).document,n=e.defaultView,t=window.top;n.jpxOnReady=function(e){var r=n.jPAM||t.jPAM||{},o=function(){var r=(n.jPAM||t.jPAM||{}).subscribe(\"adReady\",function(n){r(),e(n[1])})};r.initialized?o():(r.cmd=r.cmd||[],r.cmd.push(o))},n.jpxApi=o(n),e.open(),e.write('<html><head><style type=\"text/css\">html,body{padding:0;margin:0;background-color:transparent;overflow:hidden;}</style></head><body>'+(r||\"\")+\"</body>\"),e.close()}:r&&(d.innerHTML=r),n&&n.appendChild)return n.appendChild(d)&&d;if(void 0===n[1])return n[0].appendChild(d)&&d;var c=n[0].children[n[1]];return n[0].insertBefore(d,c),d}var a,p=window.top.document,d=\"id\",c=\"div\",l=[\"addSheet\",\"close\",\"hideCloseButton\",\"on\",\"mute\",\"play\",\"pause\",\"unmute\",\"getPercentageVisibility\",\"enableTransparency\",\"removeBoxShadow\",\"changeCounterColor\",\"hideLayer\",\"pushDown\",\"pushUp\"];return i(\"style\",p.body,[[\"type\",\"text/css\"]],\".jpx-off-screen{position: fixed !important; left: -9999px; top: -9999px;}\"),i(\"iframe\",i(c,i(c,i(c,i(c,function(e){try{e=!!e&&JSON.parse(e)}catch(n){e=!1}var n,t;if(!e)return p.body;for(;!n&&e.length;){var r=((t=e.shift())[0]||\"\").match(/:index\\((\\d+)\\)/i);if(r){t[0]=t[0].split(\":index(\")[0];var o=p.querySelectorAll(t[0].split(\":index(\")[0]);n=parseInt(r[1])>-1&&o.length>parseInt(r[1])?o[r[1]]:o[o.length-1]}else n=p.querySelector(t[0])}if(!n)return p.body;var i,a,d=n;switch(t[1]){case\"before_sibling\":d=n.parentNode,a=(i=Array.prototype.slice.call(d.children)).indexOf(n);break;case\"after_sibling\":d=n.parentNode,a=(i=Array.prototype.slice.call(d.children)).indexOf(n),a+=1;break;case\"before\":d=n.parentNode,d=(i=Array.prototype.slice.call(d.children))[a=i.indexOf(n)],a=0}return[d,a]}(r),[[\"class\",\"jpx-off-screen\"],[d,\"pjx-wrapper-\"+n]]),[[d,\"pjx-container-\"+n]]),[[d,\"pjx-content-\"+n]]),[[d,\"pjx-ad-\"+n]]),[[d,\"pjx-adframe-\"+n]],e),a&&a(),n}))</script>"}]},"pass":{"34364":false},"deals":{}}
```
Are you ok with that or we need to change something here?

Reply:
In this case everything looks good here, thanks for sharing a live bid response